spark/src/test/java/org/apache/iceberg/spark/source/TestSparkTableUtil.java

```diff
@@ -200,4 +209,48 @@ public void testImportAsHiveTable() throws Exception {
     long count2 = spark.read().format("iceberg").load(DB_NAME + ".test_partitioned_table").count();
     Assert.assertEquals("three values ", 3, count2);
   }
+
+  @Test
+  public void testImportWithNameMapping() throws Exception {
```

Comment:
Can you add a test for name mapping with the vectorized read path? Just set the property on the table and it should take that path.

Reply:
Done, also updated the vectorized reader builder. Thanks for catching this